doc/specs/definitions.yaml

```diff
@@ -38,6 +38,95 @@ definitions:
     type: array
     items:
       $ref: "#/definitions/Profile"
+  Activity:
+    allOf:
+      - $ref: "#/definitions/Profile"
+    properties:
+      activity_likes:
```

Comment:
Måske de bare skulle hedde likes, posts osv. istedet for activity_likes, da alt er defineret under activity i forvejen?